src/lib/uniforms-surfnet/src/SelectField.tsx

```diff
@@ -103,12 +103,12 @@ function Select({
                     labelAppend={<EuiText size="m">{description}</EuiText>}
                     error={showInlineError ? errorMessage : false}
                     isInvalid={error}
-                    id={id}
+                    id={`select-formrow-${name}`}
```

Comment:
This seems weird: what's in the id var itself?

Reply:
Nothing useful :) 

When I revert this line and run the SelectField tests, 2 of them fail showing different deltas.

The first testcase is `"<SelectField> - renders a select with correct value (model)"`
```
    -           "id": "select-formrow-x-row",
    +           "id": "uniforms-0000-000k-row",


    -                   "for": "select-formrow-x",
    -                   "id": "select-formrow-x-label",
    +                   "for": "uniforms-0000-000k",
    +                   "id": "uniforms-0000-000k-label",
```
`-x-` is the name parameter set on the component under test `<SelectField name="x" />`
So previously the id was generated by uniforms and thus random.


The second testcase is `"<SelectField> - renders a select with correct value (as uniqueItem list child)"`
```
    -                         "id": "select-formrow-x.0-row",
    +                         "id": "list-autofield-x.0-row",


    -                                 "for": "select-formrow-x.0",
    -                                 "id": "select-formrow-x.0-label",
    +                                 "for": "list-autofield-x.0",
    +                                 "id": "list-autofield-x.0-label",
```
Again `-x-` is the name parameter set on the component, `<ListField name="x" />`.
The previous id here was also "uniforms" - it now shows "list-autofield" because of the change I did in [ListItemField.tsx](https://github.com/workfloworchestrator/orchestrator-core-gui/pull/112/files#diff-0c7821dfcc113e62ac5851d2a4116b68835edc41e49fd98c038c509fbb51f7b5)

Reply:
Reverted and solved ID problem in testcode